doc/api.rst

```diff
@@ -862,6 +862,289 @@ Find the last 10 commands signed by an investigator identified by name.
 
 	/api/v1/search?investigatorname=%25bob%25smith%25&limit=10&type=command
 
+GET /api/v1/loader
+~~~~~~~~~~~~~~~~~~
+* Description: Returns the details of a particular loader instance
+* Parameters:
+	- `loaderid`: ID of loader instance to return
+* Authentication: X-PGPAUTHORIZATION
+* Response Code: 200 OK
+* Reponse: Collection+JSON
+
+.. code:: json
+
+        {
+            "collection": {
+                "error": {},
+                "href": "http://api.mig.example.net:1664/api/v1/loader?loaderid=12",
+                "items": [
+                    {
+                        "data": [
+                            {
+                                "name": "loader",
+                                "value": {
+                                    "AgentName": "corbomite.internal",
+                                    "Enabled": true,
+                                    "ID": 12,
+                                    "Key": "",
+                                    "LastSeen": "2016-05-17T14:10:03.041024-05:00",
+                                    "Name": "corbomite.internal"
```

Comment:
Are those keys really capitalized? If so, can they be lowercased to be consistent with other API responses?